packages/unit/test/widget/pageable_list_test.dart

```diff
@@ -79,4 +98,28 @@ void main() {
       expect(currentPage, equals(5));
     });
   });
+
+  test('PageableList resize parent', () {
+    testWidgets((WidgetTester tester) {
+      tester.pumpWidget(new Container());
+      currentPage = null;
+      itemsWrap = true;
+
+      tester.pumpWidget(buildFrame());
+      expect(currentPage, isNull);
+      pageRight(tester);
+      expect(currentPage, equals(5));
+
```

Comment:
Here we expect the PageableList to be 800x600. I think it would be worth checking that.